docs/examples/pydata.md

```diff
@@ -91,10 +91,21 @@ data = xr.DataArray(
 data
 ```
 
+## ipyleaflet
+
+`ipyleaflet` is a **Jupyter**/**Leaflet** bridge enabling interactive maps in the Jupyter notebook environment. this demonstrate how you can integrate maps in your documentation.
+
+```{code-cell}
+from ipyleaflet import Map, basemaps
+
+# display a map centered on France
+m = Map(basemap=basemaps.Esri.WorldImagery,  zoom=5, center=[46.21, 2.21])
+m
```

Comment:
```suggestion
```

Reply:
now it's displayed 0 times, that's weird